src/bindings.ts

```diff
@@ -0,0 +1,29 @@
+import { HexString } from '@polkadot/util/types'
+
+import { Blockchain } from './blockchain'
+import { defaultLogger, truncate } from './logger'
+
+const logger = defaultLogger.child({ name: 'binding' })
+
+export const setupBindings = (chain: Blockchain) => {
+  global.getStorage = async function (blockHash: HexString, key: HexString) {
```

Comment:
can we put those under a namespace to reduce the chance of conflicts?
something like `global._chopsticks_binding_ = { getStorage() {}, getPrefixKeys() {} }`

Reply:
Yes good idea